lib/cli/src/Cardano/CLI.hs

```diff
@@ -1263,6 +1399,20 @@ walletStyleOption = option (eitherReader fromTextS)
                 ++ ult
          formatEnglishEnumerationRev xs = intercalate ", " (reverse xs)
 
+keyArgument :: Parser Text
+keyArgument = T.pack <$> (argument str (metavar "HEX-XPRV"))
+
+pathOption :: Parser [DerivationIndex]
+pathOption = option (eitherReader f) $
+    mempty
+    <> long "path"
+    <> metavar "DER-PATH"
+    <> help "Derivation path e.g. \"44'/1815'/0'/0\""
+    <> value ([DerivationIndex 0x80000000])
+  where
+    f :: String -> Either String [DerivationIndex]
+    f s = left getTextDecodingError $ mapM fromText $ T.splitOn "/" (T.pack s)
```

Comment:
Is the empty derivation path handled anywhere?

This seems to overlap the functionality of `instance FromText DerivationPath`.

Reply:
> This seems to overlap the functionality o
😮 Indeed, thanks!

> Is the empty derivation path handled anywhere?

It is currently handled somewhat implicitly by `T.splitOn "/"` returning `""`, which then results in the `DerivationIndex` failing with _An empty string is not a derivation index!_

Perhaps it could be desirable to allow empty paths (returning the input key), not sure. In-doubt I thought it would probably be a user error, and the CLI erroring would be desirable.